src/lib.rs

```diff
@@ -168,32 +487,124 @@ impl Child {
         }
     }
 
-    pub fn kill(&self) -> Result<()> {
+    /// Returns process identifier associated with this child.
+    ///
+    /// Equivalent to the [`GetProcessId`][get-process-id] function.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use CreateProcessW::Command;
+    ///
+    /// let mut command = Command::new("notepad.exe");
+    ///
+    /// if let Ok(child) = command.spawn() {
+    ///     match child.id() {
+    ///         Ok(id) => println!("Child's ID is {}", id),
+    ///         Err(err) => println!("Cannot get child's ID"),
+    ///     }
+    /// } else {
+    ///     println!("notepad didn't start");
+    /// }
+    /// ```
+    ///
+    /// [get-process-id]: https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-getprocessid
+    pub fn id(&self) -> Result<u32> {
         unsafe {
-            let res = TerminateProcess(self.process_information.hProcess, 0);
+            let process_id = GetProcessId(self.process_information.hProcess);
 
-            if res.as_bool() {
-                Ok(())
-            } else {
-                Err(Error::KillFailed(GetLastError().0))
+            match process_id {
+                0 => Err(Error::GetProcessIdFailed(GetLastError().0)),
+                _ => Ok(process_id),
             }
         }
     }
 }
 
+use windows::Win32::Foundation::CloseHandle;
+
+// This function isn't really necessary, but avoid code repetition
+unsafe fn close_handles(process_info: &PROCESS_INFORMATION) {
+    CloseHandle(process_info.hProcess);
+    CloseHandle(process_info.hThread);
+}
+
+/// Describe the result of a process after it has terminated.
+///
+/// This struct is used to represent the exit status or other termination of a
+/// child process. Child processes are created via the [`Command`] struct and
+/// their exit status is exposed through the [`status`][Command::status]
+/// method, or the [`wait`][Child::wait] method of a [`Child`] process.
 pub struct ExitStatus(u32);
```

Comment:
**Always** impl Debug.

```suggestion
#[derive(Debug)]
pub struct ExitStatus(u32);
```

Also try to implement the same traits if possible:

![image](https://user-images.githubusercontent.com/3018448/142426698-1e218b6b-ffdf-49e0-b0e5-95413f9b25dc.png)